modules/flight_interface/flight_interface_worker.py

```diff
@@ -15,6 +16,7 @@ def flight_interface_worker(
     baud_rate: int,
     period: float,
     output_queue: queue_proxy_wrapper.QueueProxyWrapper,
+    odometry_queue: queue_proxy_wrapper.QueueProxyWrapper,
```

Comment:
Rename to something more descriptive (e.g. `most_recent_odometry_queue` ). Both queues hold odometry data.